content/en/universal_service_monitoring/setup.md

```diff
@@ -949,7 +949,6 @@ Universal Kafka Monitoring is available in <strong>Private beta</strong>.
 <ul role="list">
   <li>Producers and consumers require Linux Kernel version 5.2 or later.</li>
   <li>Producers and consumers must be interfacing with Kafka <strong>without</strong> TLS.</li>
-  <li>Datadog recommends Helm with Datadog chart version 2.26.2 or later. The chart version is defined in the <a href="https://github.com/DataDog/helm-charts/blob/main/charts/datadog/Chart.yaml">`chart.yaml`</a> file of the dd-agent.</li>
```

Comment:
also, remove "Universal" from the title and explanation

Reply:
Done